### PR TITLE
Add metview-python

### PR DESCRIPTION
**Pull request by iainrussell**
Purpose: to add a new feedstock, metview-python, following the example of magics-python, also from ECMWF.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by iainrussell** [comment]
@sandorkertesz - are you willing to be listed as a maintainer of metview-python?

**Comment by sandorkertesz** [comment]
Yes, thank you.

File: recipes/metview-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  commands:
+    - test -f $(python -c "import site; print(site.getsitepackages()[0])")/metview/__init__.py  # [not win]
+
+about:
+  home: https://github.com/ecmwf/metview-python
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - PKG-INFO
```
Comment (dopplershift) [inline recipes/metview-python/meta.yaml:40]: This should be `LICENSE` (based on the PyPI tarball)

Comment (iainrussell) [reply recipes/metview-python/meta.yaml]: Quite right, I've fixed it now. Thanks Ryan (@dopplershift) !

**Comment by iainrussell** [comment]
Thanks @dopplershift ! I thanked you yesterday too, but the comment seems to have been lost - probably forgot to press 'Comment' or something! I can confirm that the new package is working nicely!

**Pull request merged**